RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -637,6 +636,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      }
    }
 
+   // muon arbitration
+   fillArbitrationInfo( outputMuons.get() );
+   arbitrateMuons( outputMuons.get(), caloMuons.get() );
```

Comment:
should this always be done or should there be an option to not do it (for studies etc).
Recall that this module is used also to make muons as seeds for inside-out tracking iteration.
I can guess that for that case it's OK to have a larger number of options.